.config/webpack.config.dev.js

```diff
@@ -0,0 +1,41 @@
+const { helpers, externals, presets } = require( '@humanmade/webpack-helpers' );
+
+const { choosePort, cleanOnExit, filePath } = helpers;
+
+// Clean up manifests on exit.
+cleanOnExit( [
+	filePath( 'assets/dist/asset-manifest.json' ),
+] );
+
+module.exports = choosePort( 8080 ).then( ( port ) => [
+	presets.development( {
+		name: 'editor',
+		externals,
+		devServer: {
+			port,
+		},
+		entry: {
+			editor: filePath( 'assets/src/editor.js' ),
+		},
+		output: {
+			chunkFormat: 'array-push',
+			path: filePath( 'assets/dist' ),
+			publicPath: `http://localhost:${port}/annual-report/`,
+		}
+	} ),
+	presets.development( {
```

Comment:
This is causing issues in getting the environment going for me. Webpack complains of the same port being used twice:

![image](https://github.com/wikimedia/wikimedia-wordpress-annual-report-plugin/assets/136342/4f2ccd7d-df7e-4a7e-9e4a-6469fa508821)

If I remove this second environment config, I get a successful start up:

![image](https://github.com/wikimedia/wikimedia-wordpress-annual-report-plugin/assets/136342/29f32f24-00bc-450a-a31f-a236b6e24a92)

That said, trying to load in the browser initially gives me "Cannot GET /annual-report/". I haven't dug into it any further yet :)